01/ronaldokun/wordvalue.py

```diff
@@ -0,0 +1,34 @@
+import re
+
+from data import DICTIONARY, LETTER_SCORES
+
+char = re.compile(r"[A-Z]{1}")  # I want to match individual UPPER CASE LETTERS
+
+
+def load_words() -> list:
+    """Load dictionary into a list and return list"""
+    with open(DICTIONARY, "r") as words:
+        return words.read().splitlines()
```

Comment:
do not use return within a context manager `with`. Athough the lock of the file is released in any case, you cannot easily catch exceptions

Reply:
I didn't know that. I've used this many times. I'll keep in mind from now on. Thank you!